time_tracker_api/time_entries/time_entries_repository.py

```diff
@@ -233,6 +235,52 @@ def find_all_v2(
         function_mapper = self.get_mapper_or_dict(mapper)
         return list(map(function_mapper, result))
 
+    def get_last_entry(
+        self,
+        owner_id: str,
+        event_context: EventContext,
+        visible_only=True,
+        mapper: Callable = None,
+    ):
+        query_builder = (
+            CosmosDBQueryBuilder()
+            .add_sql_where_equal_condition({'owner_id': owner_id})
+            .add_sql_order_by_condition('end_date', Order.DESC)
+            .add_sql_limit_condition(1)
+            .add_sql_offset_condition(1)
```

Comment:
Could you please evaluate if you really need the offset condition.

Reply:
It is needed to take the last entry instead of getting the running time entry. We evaluate your suggestion of excluding the running time entry id, but it was necessary to create other methods to carry out that. 
So, we thought it was better to use a function that we already have.